### PR TITLE
Add the ability to set stop_loss OTO orders.

### PR DESCRIPTION
**Pull request by husafan**
Also support bracket orders when stop_loss and take_profit are set.

### CODE REVIEW

File: src/args.rs [hunk 1]
```diff
@@ -18,7 +18,6 @@ use structopt::StructOpt;
 use uuid::Error as UuidError;
 use uuid::Uuid;
 
-
```
Comment (d-e-s-o) [inline src/args.rs]: Can you please remove any spurious changes from the pull request? Thanks.

File: src/args.rs [hunk 2]
```diff
@@ -335,6 +322,15 @@ pub struct SubmitOrder {
   /// Create a one-triggers-other order with the given take-profit price.
   #[structopt(long)]
   pub take_profit_price: Option<Num>,
+  /// Create a one-triggers-other order with the given stop_price under the
```
Comment (d-e-s-o) [inline src/args.rs]: This documentation seems to need adjustment, now that bracket orders are supported. Same comment to the argument above and below.

File: src/args.rs [hunk 3]
```diff
@@ -335,6 +322,15 @@ pub struct SubmitOrder {
...
+  /// stop_loss dvanced order leg.
+  #[structopt(long)]
+  pub stop_loss_stop_price: Option<Num>,
+  /// Create a one-triggers-other order with the given limit_price under the
+  /// stop_loss dvanced order leg. Note that this option can only be used in
```
Comment (d-e-s-o) [inline src/args.rs]: What is `dvanced`?

File: src/main.rs [hunk 4]
```diff
@@ -883,6 +875,16 @@ async fn order_submit(client: Client, submit: SubmitOrder) -> Result<()> {
 
   let type_ = determine_order_type(&limit_price, &stop_price);
   let take_profit = take_profit_price.map(order::TakeProfit::Limit);
+  let stop_loss = if stop_loss_stop_price.is_some() && stop_loss_limit_price.is_some() {
+    Some(order::StopLoss::StopLimit(
+      stop_loss_stop_price.unwrap(),
+      stop_loss_limit_price.unwrap(),
```
Comment (d-e-s-o) [inline src/main.rs]: Would you mind rewriting the code so that we do not require `unwrap`s? E.g., using a `match` expression,

**Review by d-e-s-o — CHANGES_REQUESTED** [review]
Thanks for the pull request! Looks mostly good, I just have a few comments.

File: src/args.rs (hunk 1 above)
Comment (husafan) [reply src/args.rs]: Done.

**Comment by husafan** [comment]
Thank you for the quick response. Let me know if there are any other changes you'd like to see.

**Comment by d-e-s-o** [comment]
Looks great now. I've merged the change with some minor modifications. Thanks for your contribution!

**Pull request closed without merging**